assets/src/edit-story/components/carousel/karma/popups.karma.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+
+describe('Popup Menus - Help Center and Checklist', () => {
+  let fixture;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    fixture.setFlags({ enableChecklistCompanion: true });
+    await fixture.render();
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should start with both popups closed', () => {
+    const { helpCenterToggle, checklistToggle } = fixture.editor.carousel;
+
+    expect(helpCenterToggle).toBeDefined();
+    expect(checklistToggle).toBeDefined();
```

Comment:
💭 I think this should check to see that the dialogs do not exist since the toggles should always exist. If the `getByRole` is updated to `queryByRole` then it could return `null` if there is no dialog rendered.